gadget/gadget.go

```diff
@@ -129,6 +129,9 @@ type Volume struct {
 	Structure []VolumeStructure `yaml:"structure" json:"structure"`
 	// Name is the name of the volume from the gadget.yaml
 	Name string `json:"-"`
+	// MinSize is the minimal size required by this volume, as implicitly
+	// defined by the size structures.
+	MinSize *quantity.Size `json:"-" yaml:"-"`
```

Comment:
coudn't this be a method with a cache in the struct instead? it would avoid having to remember to set this in tests?

Reply:
I actually like having it in the structure as in the end it is implicitly defined - and it is even something we might want some day in gadget.yaml if, say, it is convenient to leave an empty space at the end of the volume. Having to set it in tests helps with testing the calculation of this quantity.